### PR TITLE
Chatroom messages

### PR DESCRIPTION
**Pull request by stshayna**
Please review Requests#show page:
- added in item details at the top (distance to be added in a separate branch)
- added "Recommendations" inside messages box (otherwise the messages box will be too small

Let me know if there's any questions :)

Screenshot 2022-06-07 at 11 20 56 PM

### CODE REVIEW

**Comment by MarlAndre** [comment]
I think it will be better to copy paste some of the COVID safety recommendation. Otherwise, it looks very good !

**Comment by stshayna** [comment]
@MarlAndre I've changed the wording as follows:
Trying to keep the entire page not too wordy so I added two more recommendations.
 
Screenshot 2022-06-08 at 10 27 50 AM

**Comment by MarlAndre** [comment]
Can you put everything on the same line ?

**Comment by MarlAndre** [comment]
Also, we should be able to close the notice recommendation.
You can look at the code I did on index page, line 17 to 29.
I think this code should do the trick : 
`    <span class="closebtn" onclick="this.parentElement.style.display='none';">&times;</span>
`

**Comment by stshayna** [comment]
@MarlAndre yeah, but i don't wish to close the recommendations so receiver/giver can scroll up to see the recommendations before they meet...do you think it's a good idea?

**Comment by stshayna** [comment]
updated item details as requested:

Screenshot 2022-06-08 at 11 17 52 AM

**Comment by stshayna** [comment]
Screenshot 2022-06-08 at 11 24 03 AM

The sentence can be in one line if the font-size is smaller but may be too small.

**Comment by MarlAndre** [comment]
> @MarlAndre yeah, but i don't wish to close the recommendations so receiver/giver can scroll up to see the recommendations before they meet...do you think it's a good idea?

If it's the same logic that the one in index page, it will reappear every time you go back to the conversation :)

**Comment by stshayna** [comment]
@MarlAndre I've added the close button as requested. Feel free to change anything as preferred!
Screenshot 2022-06-08 at 8 04 03 PM
